ui/org.eclipse.pde.junit.runtime/META-INF/MANIFEST.MF

```diff
@@ -12,5 +12,5 @@ Export-Package: org.eclipse.pde.internal.junit.runtime;x-internal:=true
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Import-Package: org.eclipse.ui.testing;resolution:=optional
-DynamicImport-Package: org.junit.platform.engine
+DynamicImport-Package: org.junit.platform.engine;version="[1.14.0,2.0.0)"
```

Comment:
I have to admit that I didn't knew that dynamically imported packages can have a version(range) too.
In general I wonder if we should better convert this to an optional package-import?
IIRC dynamic imports still create wires at runtime, but only on demand/dynamically. But as a test-runtime usually doesn't change I'm think if an ordinary (optional) import would be fine.
Rereading my comment from https://github.com/eclipse-pde/eclipse.pde/pull/1047#discussion_r1470375700, I think there was no greater meaning in it.
And we already have an optional requirement on `org.eclipse.ui`.

Reply:
The difference between an optional and a dynamic import is that the optional one is resolved at the time the bundle is resolved, and the dynamic one is resolved at the time you try to access that package (+dynamic supports wildcards)